go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/fullstorydev/grpcurl v1.7.0
 	github.com/go-bindata/go-bindata v0.0.0-20191126083508-8639be0519b3 // indirect
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.1
```

Comment:
It doesn't look like you are actually doing anything that requires any updates to `go.mod`.

Due to compatibility issues between 1.3.5 and 1.4.1, I'd rather not update this repo just yet. (You can, of course, use v1.4.1 in our own module that uses grpcui; but I don't want to foist the new version on _every_ user yet.)